scripts/env.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
Would be best to avoid bashisms and use standard `/bin/sh`, no?

Reply:
Also `set -e` all these please.

Reply:
added set -e